src/ubuntu/16.04/x64/cross/hooks/post-build

```diff
@@ -0,0 +1 @@
+../../../build-scripts/build-rootfs-cleanup.sh
```

Comment:
Path doesn't seem correct, no ending line feed, no "shebang" - in general this should be consistent with the other post/pre build scripts.